easybuild/tools/module_naming_scheme/hierarchical_mns.py

```diff
@@ -54,13 +54,15 @@
     # required for use of iccifort toolchain
     'icc,ifort': ('intel', '%(icc)s'),
     'iccifort': ('intel', '%(iccifort)s'),
+    'intel-compilers': ('intel', '%(intel-compilers)s'),  # OneAPI compilers
     # required for use of ClangGCC toolchain
     'Clang,GCC': ('Clang-GCC', '%(Clang)s-%(GCC)s'),
     # required for use of gcccuda toolchain, and for CUDA installed with GCC toolchain
     'CUDA,GCC': ('GCC-CUDA', '%(GCC)s-%(CUDA)s'),
     # required for use of iccifortcuda toolchain
     'CUDA,icc,ifort': ('intel-CUDA', '%(icc)s-%(CUDA)s'),
     'CUDA,iccifort': ('intel-CUDA', '%(iccifort)s-%(CUDA)s'),
+    'CUDA,intel-compilers': ('intel-CUDA', '%(intel-compilers)s-%(CUDA)s'),  # OneAPI compilers
```

Comment:
I'm going to remove this since we are about to unify iccifort and `iccifortcuda` (so I have no toolchain to test this on, and we don't plan on having one).